src/utils/jwt.ts

```diff
@@ -88,8 +88,34 @@ export class JWT {
    *
    * const token = Mux.JWT.sign('some-playback-id', { keyId: 'your key id', keySecret: 'your key secret' });
    * // Now you can use the token in a url: `https://stream.mux.com/some-playback-id.m3u8?token=${token}`
+   *
```

Comment:
We should probably expand the `TypeClaim` enum above to also handle spaces, as well as real-time viewer counts (see https://www.mux.com/blog/surface-real-time-view-and-unique-viewer-counts-in-your-video-player).

Reply:
Also, I'm assuming that signing a JWT for viewer counts can use the existing `JWT.signPlaybackId` method, assuming we document the additional `options.type=` parameter

Reply:
Only added stats here because I wouldn't think we would want `sign` to be able to sign spaces as `sign` proxies `signPlaybackId`